Makefile

```diff
@@ -6,7 +6,7 @@ NOOBAA_TAG?="noobaa"
 NOOBAA_BASE_TAG?="noobaa-base"
 SUPPRESS_LOGS?=""
 NO_CACHE?=""
-
+CPUS?=$(shell nproc --ignore=1)
```

Comment:
There is no nproc for macOS
so we can find a way to detect the os and then if we find a way we can do something like:
if Linux shell nproc --ignore=1
if macOS shell alias nproc='sysctl -n hw.ncpu' and shell nproc --ignore=1

Reply:
Perhaps we should split the Makefile differently - as I assume there's no podman on Mac either? (not sure) - one Makefile for CI and Linux, and another for general use.

I'm not too happy about it - because I strongly prefer having a single Makefile... 
ARGH. Need to think about it some more.